Azure-Samples/ms-identity-javascript-tutorial
# Updating packages

# Pull Request

## Purpose

The purpose of this pull request is to address the package issues being identified by `npm audit`. Fixes resolve issues for development and production builds.

* ...

## Does this introduce a breaking change



```console
    [ ] Yes
    [x] No
```

## Pull request type

What kind of change does this Pull Request introduce?



```console
    [ ] Bugfix
    [ ] Feature
    [ ] Code style update (formatting, local variables)
    [ ] Documentation content changes
    [x] Other... Please describe: Package update
```

## How to test

* Get the code

```console
    git clone [repo-address]
    cd [repo-name]
    git checkout [branch-name]
    npm install
```

* Test the code



```console
* Run code as described in README. There should be no changes.
```

## What to check

ex: verify that the following are valid:

* No breaking changes are added from these changes.
* Running `npm audit --production` and `npm audit` shows `0 vulnerabilities` in all folders with a `package.json` file.

## Other Information
